tools/i18n_autogen.py

```
@@ -0,0 +1,195 @@
+        return key
+    h = hashlib.sha1(s.encode('utf-8')).hexdigest()[:8]
+    return 'k_' + h
+
+# collect literals per source file
+files = list(SRC.rglob('*.py'))
+per_file_literals = {}  # path -> {'str': set(), 're': set()}
+for f in files:
+    try:
+        txt = f.read_text(encoding='utf-8')
+    except Exception:
+        continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)